<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1225</width>
    <height>430</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="Photo_Original">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>10</y>
      <width>401</width>
      <height>351</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>360</y>
      <width>441</width>
      <height>41</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="salvar">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>20</height>
        </size>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="text">
        <string>salvar</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="saida">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>20</height>
        </size>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="text">
        <string>sair</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </widget>
   <widget class="QScrollArea" name="scrollArea">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>441</width>
      <height>411</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="sizeAdjustPolicy">
     <enum>QAbstractScrollArea::AdjustIgnored</enum>
    </property>
    <property name="widgetResizable">
     <bool>false</bool>
    </property>
    <widget class="QWidget" name="scrollAreaWidgetContents">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>445</width>
       <height>659</height>
      </rect>
     </property>
     <widget class="QPushButton" name="inserir_imagem">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>421</width>
        <height>41</height>
       </rect>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>Inserir Imagem</string>
      </property>
     </widget>
     <widget class="QPushButton" name="flip_vertical">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>60</y>
        <width>201</width>
        <height>41</height>
       </rect>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="text">
       <string>Flip Vertical</string>
      </property>
     </widget>
     <widget class="QPushButton" name="flip_horizontal">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>110</y>
        <width>201</width>
        <height>41</height>
       </rect>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="text">
       <string>Flip Horizontal</string>
      </property>
     </widget>
     <widget class="QPushButton" name="brilho">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>160</y>
        <width>101</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Ajusta Brilho</string>
      </property>
     </widget>
     <widget class="QPushButton" name="contraste_botao">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>210</y>
        <width>101</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Ajusta Contraste</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="numBrilho">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>160</y>
        <width>91</width>
        <height>41</height>
       </rect>
      </property>
      <property name="minimum">
       <number>-255</number>
      </property>
      <property name="maximum">
       <number>255</number>
      </property>
     </widget>
     <widget class="QDoubleSpinBox" name="valor_contraste">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>210</y>
        <width>91</width>
        <height>41</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="negativo_botao">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>260</y>
        <width>201</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Calculo Negativo</string>
      </property>
     </widget>
     <widget class="QPushButton" name="equalize_botao">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>310</y>
        <width>201</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Equalizar</string>
      </property>
     </widget>
     <widget class="QPushButton" name="rotate_esq">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>360</y>
        <width>101</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;-</string>
      </property>
     </widget>
     <widget class="QPushButton" name="rotate_dir">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>360</y>
        <width>91</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>-&gt;</string>
      </property>
     </widget>
     <widget class="QPushButton" name="Z_in">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>310</y>
        <width>201</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Zoom In</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="z_width">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>260</y>
        <width>101</width>
        <height>41</height>
       </rect>
      </property>
      <property name="minimum">
       <number>1</number>
      </property>
     </widget>
     <widget class="QSpinBox" name="z_height">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>260</y>
        <width>91</width>
        <height>41</height>
       </rect>
      </property>
      <property name="minimum">
       <number>1</number>
      </property>
     </widget>
     <widget class="QPushButton" name="Z_out">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>210</y>
        <width>201</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Zoom Out</string>
      </property>
     </widget>
     <widget class="QPushButton" name="quantizar">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>160</y>
        <width>101</width>
        <height>41</height>
       </rect>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="text">
       <string>Quantização</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="spinBox">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>160</y>
        <width>91</width>
        <height>41</height>
       </rect>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="maximum">
       <number>255</number>
      </property>
     </widget>
     <widget class="QPushButton" name="geraHistograma">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>110</y>
        <width>201</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Gerar Histograma</string>
      </property>
     </widget>
     <widget class="QPushButton" name="tons_de_cinza">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>60</y>
        <width>201</width>
        <height>41</height>
       </rect>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="text">
       <string>Tons de Cinza</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>360</y>
        <width>201</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Matching</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>920</x>
      <y>0</y>
      <width>221</width>
      <height>21</height>
     </rect>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="lineWidth">
     <number>2</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>1000</x>
      <y>10</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Convoluções</string>
    </property>
   </widget>
   <widget class="Line" name="line_2">
    <property name="geometry">
     <rect>
      <x>920</x>
      <y>40</y>
      <width>221</width>
      <height>21</height>
     </rect>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="lineWidth">
     <number>2</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="conv_gaus">
    <property name="geometry">
     <rect>
      <x>970</x>
      <y>60</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Gaussiana</string>
    </property>
   </widget>
   <widget class="QPushButton" name="conv_lapl">
    <property name="geometry">
     <rect>
      <x>890</x>
      <y>100</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Laplaciano</string>
    </property>
   </widget>
   <widget class="QPushButton" name="conv_passa_altas">
    <property name="geometry">
     <rect>
      <x>1050</x>
      <y>100</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Passa Altas Genérico</string>
    </property>
   </widget>
   <widget class="QPushButton" name="conv_Px">
    <property name="geometry">
     <rect>
      <x>890</x>
      <y>140</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Prewitt Hx</string>
    </property>
   </widget>
   <widget class="QPushButton" name="conv_Py">
    <property name="geometry">
     <rect>
      <x>1050</x>
      <y>140</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Prewitt Hy</string>
    </property>
   </widget>
   <widget class="QPushButton" name="conv_sx">
    <property name="geometry">
     <rect>
      <x>890</x>
      <y>180</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Sobel Hx</string>
    </property>
   </widget>
   <widget class="QPushButton" name="conv_sy">
    <property name="geometry">
     <rect>
      <x>1050</x>
      <y>180</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Sobel Hy</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="personal_00">
    <property name="geometry">
     <rect>
      <x>930</x>
      <y>230</y>
      <width>62</width>
      <height>25</height>
     </rect>
    </property>
    <property name="minimum">
     <double>-99.000000000000000</double>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="personal_01">
    <property name="geometry">
     <rect>
      <x>1010</x>
      <y>230</y>
      <width>62</width>
      <height>25</height>
     </rect>
    </property>
    <property name="minimum">
     <double>-99.000000000000000</double>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="personal_02">
    <property name="geometry">
     <rect>
      <x>1090</x>
      <y>230</y>
      <width>62</width>
      <height>25</height>
     </rect>
    </property>
    <property name="minimum">
     <double>-99.000000000000000</double>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="personal_10">
    <property name="geometry">
     <rect>
      <x>930</x>
      <y>260</y>
      <width>62</width>
      <height>25</height>
     </rect>
    </property>
    <property name="minimum">
     <double>-99.000000000000000</double>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="personal_11">
    <property name="geometry">
     <rect>
      <x>1010</x>
      <y>260</y>
      <width>62</width>
      <height>25</height>
     </rect>
    </property>
    <property name="minimum">
     <double>-99.000000000000000</double>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="personal_12">
    <property name="geometry">
     <rect>
      <x>1090</x>
      <y>260</y>
      <width>62</width>
      <height>25</height>
     </rect>
    </property>
    <property name="minimum">
     <double>-99.000000000000000</double>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="personal_20">
    <property name="geometry">
     <rect>
      <x>930</x>
      <y>290</y>
      <width>62</width>
      <height>25</height>
     </rect>
    </property>
    <property name="minimum">
     <double>-99.000000000000000</double>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="personal_21">
    <property name="geometry">
     <rect>
      <x>1010</x>
      <y>290</y>
      <width>62</width>
      <height>25</height>
     </rect>
    </property>
    <property name="minimum">
     <double>-99.000000000000000</double>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="personal_22">
    <property name="geometry">
     <rect>
      <x>1090</x>
      <y>290</y>
      <width>62</width>
      <height>25</height>
     </rect>
    </property>
    <property name="minimum">
     <double>-99.000000000000000</double>
    </property>
   </widget>
   <widget class="QPushButton" name="conv_personal">
    <property name="geometry">
     <rect>
      <x>970</x>
      <y>330</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Personalizada</string>
    </property>
   </widget>
   <zorder>scrollArea</zorder>
   <zorder>Photo_Original</zorder>
   <zorder>layoutWidget</zorder>
   <zorder>line</zorder>
   <zorder>label</zorder>
   <zorder>line_2</zorder>
   <zorder>conv_gaus</zorder>
   <zorder>conv_lapl</zorder>
   <zorder>conv_passa_altas</zorder>
   <zorder>conv_Px</zorder>
   <zorder>conv_Py</zorder>
   <zorder>conv_sx</zorder>
   <zorder>conv_sy</zorder>
   <zorder>personal_00</zorder>
   <zorder>personal_01</zorder>
   <zorder>personal_02</zorder>
   <zorder>personal_10</zorder>
   <zorder>personal_11</zorder>
   <zorder>personal_12</zorder>
   <zorder>personal_20</zorder>
   <zorder>personal_21</zorder>
   <zorder>personal_22</zorder>
   <zorder>conv_personal</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1225</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
